### PR TITLE
feat: Display QuotaBanner when disk usage has reach 80%

### PR DESCRIPTION
**Pull request by cballevre**
```
### ✨ Features

* Display QuotaBanner when disk usage has reach 80%
```

**Related PRs:**
- https://github.com/cozy/cozy-client/pull/1366

### CODE REVIEW

File: src/components/PushBanner/PushBanner.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    })
+    makeDiskInfos.mockReturnValue({
+      percentUsage
+    })
+    return render(<PushBanner />)
+  }
+
+  describe('QuotaBanner', () => {
+    it('should show quota banner when disk usage has reach 80%', () => {
+      setup(80)
```
Comment (zatteo) [inline src/components/PushBanner/PushBanner.spec.jsx:48]: extra-nit-dont-change-this-pls: 

`const setup = ({ percentUsage = 50, dismissed = false }) => {` leads to `setup({ percentUsage: 80, dismissed: true })` which is more readable than `setup(80, false)`

**Review by zatteo — APPROVED** [review]
Very nice

File: src/components/PushBanner/PushBanner.spec.jsx (hunk 1 above)
Comment (cballevre) [reply src/components/PushBanner/PushBanner.spec.jsx:48]: Noted for next time

**Comment by cballevre** [comment]
Thanks for the review 💜

**Pull request merged**